scripts/adapters.dockerfile

```diff
@@ -17,8 +17,22 @@ FROM $image
 
 COPY scripts/setup-helper-functions.sh /
 COPY scripts/setup-adapters.sh /
-RUN mkdir build && ( cd build &&  source /opt/rh/gcc-toolset-12/enable && \
-    bash /setup-adapters.sh ) && rm -rf build && dnf remove -y conda && dnf clean all
+COPY scripts/setup-centos9.sh /
+RUN mkdir build && \
+    ( \
+      cd build && \
+      source /opt/rh/gcc-toolset-12/enable && \
+      bash /setup-adapters.sh && \
+      bash /setup-centos9.sh install_cuda 12.8 \
```

Comment:
Should installing cuda be hidden behind an arg to this?

Reply:
The version maybe but the image is for our ci where we always need cuda:)